test/metabase/util/i18n_test.clj

```diff
@@ -117,15 +117,15 @@
              (fn [n] (str (i18n/deferred-trsn "{0} table" "{0} tables" n)))}]
       (testing message
         (testing "Should fall back to English if user locale & system locale are unset"
-          (mt/with-temporary-setting-values [site-locale nil])
-          (is (= "0 tables"
-                 (f 0)))
+          (mt/with-temporary-setting-values [site-locale nil]
```

Comment:
I noticed this test had incorrect nesting so I've fixed it; though I think it wasn't failing in CI anyway because `site-locale` is `nil` by default

Reply:
The changes in this file are not related to the real purpose of the PR, are they?

Reply:
No, but I noticed it while poking around this code